pkg/buses/stub/bus.go

```diff
@@ -96,7 +96,12 @@ type stubChannel struct {
 
 func (c *stubChannel) receiveMessage(message *buses.Message) {
 	for _, stubSubscription := range c.subscriptions {
-		go stubSubscription.dispatchMessage(message)
+		go func() {
+			err := stubSubscription.dispatchMessage(message)
+			if err != nil {
+				glog.Errorf("Failed to dispatch message: %v", err)
```

Comment:
Error seems a bit extreme here. The dispatch could fail because of a misconfiguration in the subscription resource or the subscriber is down. It's not necessarily the fault of the bus.

Reply:
Good point - since this is also not expected behavior how do we feel about warning?